Services/UserService/Commands/CreateUserCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿using DTO;
+using Kernel;
+using System.Net;
+using UserService.Interfaces;
+
+namespace UserService.Commands
+{
+    public class CreateUserCommand : ICreateUserCommand
+
+    {
+        public HttpStatusCode Execute(UserCredential userCredential)
```

Comment:
Do not return HttpStatusCode it is a bad practice

Reply:
Fixed